src/mlpack/methods/ann/layer/recurrent.hpp

```diff
@@ -139,6 +139,9 @@ class Recurrent
   //! Modify the gradient.
   OutputDataType& Gradient() { return gradient; }
 
+  //! Get the number of steps to backpropagate through time.
+  size_t const& Rho() const { return rho; }
+
```

Comment:
This is fine, however, little can be known by getting the ```rho``` alone. I think obtaining the other constructor parameters would be useful too.

Reply:
So, there are 4 other variables in the constructor, namely ```const StartModuleType& start```, ```const InputModuleType& input```, ```const FeedbackModuleType& feedback``` and ```const TransferModuleType& transfer```. I did not write getter functions for them because the data types that will be returned in that case are not usually returned in any other layers. Most probably this is because, these modules are pushed into the network and then returned via the getter for network. Let me know if I should still be returning them via getters and if so, what should be the style I need to follow!

Reply:
Yes you are right, I had overlooked that. No need to write separate getters for these since we have a getter for the network itself.